### PR TITLE
Port module product_by_supplier to 8.0

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

**Comment by lmignon** [comment]
I don't understand why travis fails with   INCLUDE="purchase_requisition_bid_selection,purchase_requisition_auto_rfq,purchase_requisition_auto_rfq_bid_selection" LINT_CHECK="0" ...

File: product_by_supplier/README.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product by supplier info
+========================
+
+This module categorizes each product item by supplier.It allows for users to be able to view a compiled list of products supplied by the supplier.
```
Comment (pedrobaeza) [inline product_by_supplier/README.rst]: Put a space between `...supplier.It allows...`

Comment (pedrobaeza) [inline product_by_supplier/README.rst]: You can add that this info is the corresponding to the supplier list on each product.

File: product_by_supplier/README.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* Click on install button
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/purchase-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/purchase-workflow/issues/new?body=module:%20product_by_supplier_ext%0Aversion:%200.1%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline product_by_supplier/README.rst]: version should be 8.0, not 0.1

File: product_by_supplier/__openerp__.py [hunk 3]
```diff
@@ -25,36 +25,18 @@
     'version': '1.0.1',
```
Comment (pedrobaeza) [inline product_by_supplier/__openerp__.py]: Use 8.0.1.0 according new version number proposal

**Comment by pedrobaeza** [comment]
To be consistent with existing menu, you can call it "Products by suppliers" (in plural)

**Comment by pedrobaeza** [comment]
Little nickpitckings, but tested on runbot and :+1:

**Comment by lmignon** [comment]
@pedrobaeza Thank you for the review. Code modified according to your comments.

**Comment by pedrobaeza** [comment]
Thanks for the porting :+1:

File: product_by_supplier/views/product_supplierinfo_view.xml [hunk 4]
```diff
@@ -9,12 +9,12 @@
                     <field name="name" string="Supplier"/>
                     <field name="product_code" string="Supplier Product Code"/>
                     <field name="product_name" string="Supplier Product Code"/>
```
Comment (StefanRijnhart) [inline product_by_supplier/views/product_supplierinfo_view.xml]: Could you fix this copy paste error (unrelated to your port) and change the label for product_name to 'Supplier Product Name'?

File: product_by_supplier/README.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product by supplier info
+========================
+
+This module categorizes each product item by supplier. It allows for users to be able to view a compiled list of products supplied by the supplier.
+This information is the corresponding to the supplier list on each product.
+
+Users can also directly add new products to the supplier's list.
```
Comment (StefanRijnhart) [inline product_by_supplier/README.rst:11]: Can you mention here that the module adds a new menu item that allows the user to manage their supplier/product info?

**Comment by lmignon** [comment]
@StefanRijnhart Thank you for the review. Requested changes done.

**Comment by StefanRijnhart** [comment]
Thanks for the updates! :+1:

**Pull request merged**